src/code/qualification/getConfigurationPlan.m

```diff
@@ -0,0 +1,39 @@
+function ConfigurationPlan = getConfigurationPlan(jsonFile)
+%GETCONFIGURATIONPLAN Support function:
+% Read a .json ConfigurationPlan and output the .json structure and create
+% the folder structure for the report
+%
+%   getConfigurationPlan(jsonFile)
+%       jsonFile (string) name .json file where the information of the
+%       configuration plan is stored
+%
+
+% Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org
+
+%% --------------------------------------------------% 
+% Read .json file and output its structure
+
+try
+jsonFileContent = fileread(jsonFile);
+ConfigurationPlan = jsondecode(jsonFileContent);
+writeToReportLog('INFO',[jsonFile ' was extracted successfully'],'true');
+catch
+writeToReportLog('ERROR',[jsonFile ' could not be extracted'],'true');

```

Comment:
it would be better to pass the exception to the `writeToReportLog `function, e.g. like this:

```
catch exception
    writeToReportLog('ERROR', [jsonFile ' could not be extracted:' exception.message], 'true', exception);
end 
```